jobs/keepalived/templates/bin/post-start.erb

```diff
@@ -19,6 +19,8 @@ if [[ "$iaas_name" == "gcp" && "$IS_HA" == "true" ]]; then
   export INSTANCEIP="-instanceip=<%= spec.ip %>"
   export SFBROKERPORT="-sfbrokerport=<%= link("broker").p("internal.port") %>"
   export SFREPORTPORT="-sfreportport=9294"
+  export SFEXTERNALPORT="-sfexternalport=9292"
+  export SFDEPHOOKSPORT="-sfdephooksport=9295"
```

Comment:
Better to read from individual job properties. And default to these values

Reply:
ok.. Done!